tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -36,7 +36,11 @@ AGENT_NO_PROXY="${AGENT_NO_PROXY:-}"
 PULL_TYPE_MAPPING="${PULL_TYPE_MAPPING:-}"
 IFS=',' read -a pull_types <<< "$PULL_TYPE_MAPPING"
 
-dest_dir="/opt/kata"
+INSTALLATION_PREFIX="${INSTALLATION_PREFIX:-}"
+default_dest_dir="/opt/kata"
+dest_dir=""
+[ -n "${INSTALLATION_PREFIX}" ] && dest_dir="${INSTALLATION_PREFIX}/${default_dest_dir}" \
```

Comment:
Discussion point: `default_dest_dir` already has a leading `/`, so `dest_dir="${INSTALLATION_PREFIX}/${default_dest_dir}"` will give a double slash, but is this better (ie easier to understand) than `dest_dir="${INSTALLATION_PREFIX}${default_dest_dir}"`? I'm not sure

Reply:
I guess another discussion point is whether the use of a \<condition\> && \<then\> || \<else\> one-liner, whilst clever, is better (ie easier to understand) than simply using an if statement like?
```
if [ -n "${INSTALLATION_PREFIX}" ]; then
    dest_dir="${INSTALLATION_PREFIX}/${default_dest_dir}"
else
    dest_dir="${default_dest_dir}"
fi
```

or maybe even setting the default initially e.g.
```
dest_dir="${default_dest_dir}"
[ -n "${INSTALLATION_PREFIX}" ] && dest_dir="${INSTALLATION_PREFIX}/${default_dest_dir}" # OR if here instead
```

Sorry, I might just be in a bike shedding mood, or overthinking how difficult it is to understand for the average developer.

Reply:
Sure, I will take the more explicit approach.